### PR TITLE
Remove dpasiukevich from cluster/addons/dns OWNERS

### PR DESCRIPTION
**Pull request by dpasiukevich**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Actualise OWNERS file

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster/addons/dns/OWNERS [hunk 1]
```diff
@@ -4,9 +4,7 @@ approvers:
   - bowei
   - mrhohn
   - kl52752
-  - dpasiukevich
 reviewers:
   - bowei
   - mrhohn
   - kl52752
-  - dpasiukevich
```
Comment (aojea) [inline cluster/addons/dns/OWNERS:12]: move yourself to emeritus
```
    emeritus_approvers:
      - dpasiukevich
 ```

Comment (dpasiukevich) [reply cluster/addons/dns/OWNERS:12]: sure thing, done.

**Pull request merged**